spec/fixtures/modules/catch_errors/plans/break.pp

```diff
@@ -0,0 +1,13 @@
+plan catch_errors::break(
+  TargetSpec $nodes,
+  Array $list
+) {
+  $list.each |$elem| {
+    catch_errors() || {
+      notice($elem)
+      break()
+      notice("Out of bounds")
+    }
+  }
+  return run_task('sample::echo', $nodes, message => "Break the chain")
```

Comment:
Why does this need to run the result of running a task? Could it just return some data? Or even just trigger another `notice`?